frame/beefy/src/lib.rs

```diff
@@ -105,10 +105,10 @@ impl<T: Config> Pallet<T> {
 	}
 
 	fn change_authorities(new: Vec<T::BeefyId>, queued: Vec<T::BeefyId>) {
-		// Always issue a change if `session` says that the validators have changed.
-		// Even if their session keys are the same as before, the underlying economic
-		// identities have changed. Furthermore, the digest below is used to signal
-		// BEEFY mandatory blocks.
+		// Always issue a change on each `session`.
+		// Even if the validator session keys are the same as before, the underlying economic
+		// identities may have changed.
+		// Furthermore, the digest below is used to signal BEEFY mandatory blocks.
```

Comment:
I think I would probably remove this whole comment block now, IMO it adds more confusion. The comment we have below on the `on_new_session` method is probably enough.